Update ZenCash References

hellcatz opened the pull request:
Update ZenCash references, resolves https://github.com/ZencashOfficial/zen/issues/92

nastenko commented:
Useful fix! Indeed an error queue should be cleared before getting an error code.

dkaidalov commented:
utACK

hellcatz committed:
Update net.cpp

In reference to issue: https://github.com/ZencashOfficial/zen/issues/92
"Bitcoin Core" shows up here
https://github.com/ZencashOfficial/zen_archived/blob/master/src/net.cpp#L2294
However, latest copy of net.cpp contains "Zcash" not "ZenCash"

hellcatz committed:
Update rpcrawtransaction.cpp

Update references to ZenCash

hellcatz committed:
Update init.cpp

Update references to ZenCash

hellcatz committed:
Update rpcblockchain.cpp

Update references to ZenCash

hellcatz committed:
Update rpcnet.cpp

Update references to ZenCash

hellcatz committed:
Update rpcdump.cpp

Update references to ZenCash

hellcatz committed:
Update rpcwallet.cpp

Update references to ZenCash

hellcatz committed:
Update miner.cpp

Update references to ZenCash

hellcatz committed:
Update rpcmining.cpp

Update references to ZenCash

hellcatz committed:
Update timedata.cpp

Update references to ZenCash

hellcatz committed:
Update common.h

Update references to ZenCash
